src/humancellatlas/data/metadata/api.py

```diff
@@ -651,8 +664,8 @@ class File(LinkedEntity):
     manifest_entry: ManifestEntry
     content_description: Set[str]
 
-    def __init__(self, json: JSON, manifest: Mapping[str, ManifestEntry]):
-        super().__init__(json)
+    def __init__(self, json: JSON, metadata_manifest_entry: ManifestEntry, manifest: Mapping[str, ManifestEntry]):
+        super().__init__(json, metadata_manifest_entry)
```

Comment:
Nitpick, this seems unfortunate that both `manifest` and `metadata_manifest_entry` need to be passed into the constructor even though `manifest` contains the `metadata_manifest_entry` entry. I don't see an easy fix for this however since we don't have the key (e.g. `analysis_file_0.json`) available.

Reply:
It's ok like this.